pages/portfolio.html

```diff
@@ -5,8 +5,56 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="styles/style.css" rel="stylesheet" />
 </head>
 <body>
-  
+  <header>
+    <h5>Michelle's portfolio website</h5>
+    <nav>
+      <a href=index.html>Home</a>
+      <a href="about.html">About</a>
+      <a href="portfolio.html">Projects</a>
+    </nav>
+  </header>
+  <h2>Projects:</h2>
+  <ul class="projects">
+    <li class="project">
+      <h4>Ikonpass reservation maker bot</h4>
```

Comment:
I recommend using an h3 tag here.  In semantic html h tags are used to divide the content and are used in sequential order.